snooty/rstspec.toml

```diff
@@ -153,6 +153,14 @@ argument_type = "string"
 content_type = "string"
 example = """.. todo::"""
 
+[directive.banner]
+content_type = "block"
+options.variant = "string"
```

Comment:
This is perhaps a good use case for an [enum](https://github.com/mongodb/snooty-parser/blob/master/snooty/rstspec.toml#L4-L15)!

Reply:
Bump

Reply:
Ty for reminding 🙏